ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1206,8 +1206,127 @@
                     fingerprint: false
```

Comment:
@KMAnju-2021 Please update the testbed label if it has been verified that the multicluster job can pass on the same Kind testbed.